.github/workflows/tools_continuous_delivery.yml

```diff
@@ -34,28 +26,12 @@ on:
         type: string
```

Comment:
Do we still need dispatch if we only want continuous delivery?

Reply:
removed